tests/bismark_nomeseq.nf.test.snap

```diff
@@ -41,20 +41,6 @@
                 "bismark/coverage2cytosine/coverage/SRR389222_sub2.NOMe.GpC.cov.gz",
                 "bismark/coverage2cytosine/coverage/SRR389222_sub3.NOMe.CpG.cov.gz",
                 "bismark/coverage2cytosine/coverage/SRR389222_sub3.NOMe.GpC.cov.gz",
-                "bismark/coverage2cytosine/reports",
-                "bismark/coverage2cytosine/reports/Ecoli_10K_methylated.NOMe.CpG_report.txt.gz",
-                "bismark/coverage2cytosine/reports/Ecoli_10K_methylated.NOMe.GpC_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub1.NOMe.CpG_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub1.NOMe.GpC_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub2.NOMe.CpG_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub2.NOMe.GpC_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub3.NOMe.CpG_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub3.NOMe.GpC_report.txt.gz",
-                "bismark/coverage2cytosine/summaries",
-                "bismark/coverage2cytosine/summaries/Ecoli_10K_methylated.cytosine_context_summary.txt",
-                "bismark/coverage2cytosine/summaries/SRR389222_sub1.cytosine_context_summary.txt",
-                "bismark/coverage2cytosine/summaries/SRR389222_sub2.cytosine_context_summary.txt",
-                "bismark/coverage2cytosine/summaries/SRR389222_sub3.cytosine_context_summary.txt",
```

Comment:
lots of deletion here

Reply:
Thanks for catching this. the module wasn't publishing that's y. And I changed that based on the vscode nextflow extension. reverted back now. should be all good